dbt_rpc/contracts/rpc.py

```diff
@@ -258,7 +258,7 @@ def write(self, path: str):
 class RemoteCompileResultMixin(RemoteResult):
     raw_sql: str
     compiled_sql: str
-    node: CompileResultNode
+    node: ResultNode
```

Comment:
Will this actually affect return result? We can also just validate that in beta period on cloud

Reply:
The name of the class has changed, but nothing in it, so I wouldn't think it would affect anything.